### ISSUE TITLE
Build Functions to Read, Update, and Display in Dialog: ADDR, SHEL, and HEST

### ISSUE THREAD

**Issue by C-Love** [Hennepin-County/MAXIS-scripts#206]
We have a script called ACTIONS - Shelter Expense Verif Received that will update and note ADDR, SHEL, and HEST when a verification has been received. This is being planed to move into 'Docs Received' at some point. 

Being as there is very specific policy and procedure around updating when an address change is reported but not verified, we should have a script to gather information about a change reported, update panels, and case note. Or maybe we update the current script to work for reports and verifs.

**Comment by IlseFerris** [comment]
SHELTER DEDUCTION CHANGE NC/PC CODES- TE02.08.190

Will still need to determine if we will update ACTIONS - SHELTER EXPENSE VERIFIED or create new script.

**Comment by IlseFerris** [comment]
Reassigned from @MiKaylaH22 to @C-Love since Casey has already done background processing and mapping and has been in communication with project sponsors.

**Comment by C-Love** [comment]
Remember to add functionality for if the case should be transferred.

**Comment by C-Love** [comment]
Handle for Homeowners Insurance information

**Comment by C-Love** [comment]
Also include in the ADDR update to work for inactive cases - which is in issue #252  about updating ADDR for inactive cases. Once the functionality is done we can research where we need it.

**Comment by C-Love** [comment]
Scriptwriter testing is End User Testing

**Comment by C-Love** [comment]
These are ready to be used!!!
